pandas/io/tests/test_pytables.py

```diff
@@ -2665,23 +2665,6 @@ def test_sparse_frame(self):
         self._check_double_roundtrip(ss3, tm.assert_frame_equal,
                                      check_frame_type=True)
 
-    def test_sparse_panel(self):
```

Comment:
GitHub search says otherwise.  Also, not in the `generate_legacy_storage_files.py` file.